### PR TITLE
Completed correlation section

### PR DESCRIPTION
**Pull request by johcarter**
- Separated features by OasisLMF version
- completed correlation documentation for 1.27 and 1.28

### CODE REVIEW

File: src/sections/correlation.rst [hunk 1]
```diff
@@ -213,28 +213,60 @@ The OED field list can be specified in the oasislmf settings using the **group_i
...
+In data_settings in the model settings json, the **group_fields** parameter in version 1.15 has been changed to **damage_group_fields**. 
+
+Example:
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber"] }
```
Comment (sambles) [inline src/sections/correlation.rst:241]: Stray `}` in code block which should be removed

File: src/sections/correlation.rst [hunk 2]
```diff
@@ -327,27 +359,195 @@ The example illustrated above would be specified using:
...
+**Example 1**  
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
+    "hazard_group_fields": ["areaperil_id"] },
```
Comment (sambles) [inline src/sections/correlation.rst:427]: Stray `}` in code block which should be removed

File: src/sections/correlation.rst [hunk 3]
```diff
@@ -327,27 +359,195 @@ The example illustrated above would be specified using:
...
+
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber", "coverage_type_id"] },
+    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
```
Comment (sambles) [inline src/sections/correlation.rst:446]: Stray `}` in code block which should be removed

File: src/sections/correlation.rst [hunk 4]
```diff
@@ -327,27 +359,195 @@ The example illustrated above would be specified using:
...
+**Example 3**  
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber", "coverage_type_id"] },
+    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
```
Comment (sambles) [inline src/sections/correlation.rst:464]: Stray `}` in code block which should be removed

File: src/sections/correlation.rst [hunk 5]
```diff
@@ -327,27 +359,195 @@ The example illustrated above would be specified using:
...
+**Example 4**
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber", "coverage_type_id"] },
+    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
```
Comment (sambles) [inline src/sections/correlation.rst:494]: Stray `}` in code block which should be removed

File: src/sections/correlation.rst [hunk 6]
```diff
@@ -327,27 +359,195 @@ The example illustrated above would be specified using:
...
+
+
+``Model_settings.json``
+
+.. code-block:: JSON
+
+
+    "data_settings": {
+    "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
+    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber"] },
```
Comment (sambles) [inline src/sections/correlation.rst:523]: Stray `}` in code block which should be removed

**Review by sambles — APPROVED** [review]
Some minor fixes for the model_settings.json code snippets

**Comment by johcarter** [comment]
Thanks @sambles , I have ran all json blobs through a parser and think they are all correct now. thanks for spotting that.

**Pull request merged**